### PR TITLE
Solved the LaTeX2exp error. Closes #2027

### LINKED ISSUE #2027: latex2exp not supported
```
library(plotly)
library(latex2exp)
library(ggplot2)

p <- qplot(1, "A")+
     ylab(TeX("Formula: $\\frac{2hc^2}{\\lambda^\\beta}$"))+
     xlab(TeX("$\\alpha$"))

ggplotly(p)
```
yields following error
```
Error in unique.default(x) : unimplemented type 'expression' in 'duplicated'
```

### PR DESCRIPTION
**Pull request by moutikabdessabour**
Solved The LaTeX2exp error. for more information check #2027.

Same code now gives the correct output.
```r
library(plotly)
library(latex2exp)
library(ggplot2)

p <- qplot(1, "A")+
     ylab(TeX("Formula: $\\frac{2hc^2}{\\lambda^\\beta}$"))+
     xlab(TeX("$\\alpha$"))

ggplotly(p)
```

BTW running `devtools::test()` locally resulted in the deletion of the `tests/test_that/_snaps` folder may we should consider adding this folder to `.gitignore` ? Also great package.

### CODE REVIEW

**Comment by moutikabdessabour** [comment]
`RCMD check` still gives warning even though I do not export the modified `uniq` function. 😞

**Comment by jackparmer** [comment]
Awesome. Yes please add _snaps folder to .gitignore in a separate PR

**Review by cpsievert — CHANGES_REQUESTED** [review]
Please revert the `export(uniq)` and whitespace changes

Also, the `_snaps` folder should be checked into git, not ignored

**Comment by moutikabdessabour** [comment]
@cpsievert the whitespace changes are caused by my R linter, sorry for that.

**Comment by cpsievert** [comment]
@moutikabdessabour please revert the removal of the visual tests and add a test demonstrating the fix

**Comment by moutikabdessabour** [comment]
@cpsievert done

File: tests/testthat/helper-vdiffr.R [hunk 1]
```diff
@@ -51,12 +51,12 @@ write_plotly_svg <- function(p, file) {
...
   # strip out non-deterministic fullLayout.uid
   # TODO: if and when plotly provides an API to pre-specify, use it!
-  svg_txt <- readLines(file, warn = FALSE)
+  svg_txt <- readLines(basename(file), warn = FALSE)
   strextract <- function(str, pattern) regmatches(str, regexpr(pattern, str))
   def <- strextract(svg_txt, 'defs id=\\"defs-[[:alnum:]]+\\"')
   uid <- sub("defs-", "", strextract(def, "defs-[[:alnum:]]+"))
   svg_txt <- gsub(uid, "", svg_txt, fixed = TRUE)
-  writeLines(svg_txt, file)
+  writeLines(svg_txt, basename(file))
```
[the remark this answers is not in the document]
Comment (cpsievert) [reply tests/testthat/helper-vdiffr.R:59]: What is `write_plotly_csv`?

**Pull request merged**